app/Http/Controllers/Api/DeviceController.php

```diff
@@ -142,7 +140,13 @@ public function anyRegisterDevice(RegisterDeviceRequest $request)
             'success' => true,
             'dev_mode' => false,
             'use_gps'  => $default->is_gps_location_source ? true : false,
-            'renew'    => false //@Todo - make possible to enable Temporary. Will be used to re-check BCS projects URL.
+            'renew'    => $device->renew,
+            'user_phone' => $device->user_phone,
+            'user_phone_token' => $device->user_phone_token,
+            'user_phone_confirm' => $device->user_phone_confirm,
+            'need_phone' => $device->need_phone,
+            'need_tac' => $device->need_tac,
+            'tac_text' => \Lang::get('app.tac.default', [], $request->get('lang', $lang)) //@Todo - make administrated.
```

Comment:
What does "make administrated" mean in this context? Making the `tac_text` configurable rather than a part of the translations files?

Reply:
Translation files has default T&C text which is used now for App release. 
Later we can build a place with RTE where system administrator can update the text anytime and T&C text will be showed from it.